backend/penndata/admin.py

```diff
@@ -1,10 +1,21 @@
 from django.contrib import admin
+from django.utils.html import escape, mark_safe
 
 from penndata.models import AnalyticsEvent, Event, FitnessRoom, FitnessSnapshot, HomePageOrder
 
 
+# custom fitness room admin
+class FitnessRoomAdmin(admin.ModelAdmin):
+    def image_tag(self, instance):
+        return mark_safe('<img src="%s" height="300" />' % escape(instance.image_url))
+
+    image_tag.short_description = "Image"
```

Comment:
what do these fields do?

Reply:
short_description is like the label on the left on the admin portal. i removed the other one since mark_safe did the same thing. basically lets u render html

Reply:
got it thanks!